ui/inspectors/directory-services.reel/ldap-service.reel/ldap-service.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" type="text/css" href="ldap-service.css">
+    <script type="text/montage-serialization">
+    {
+        "owner": {
+            "properties": {
+                "element": {"#": "owner"}
+            }
+        },
+
+        "serverAddress": {
```

Comment:
To have form validation to work automatically, component and property name need to be the same (here, the component should be renamed to `server`).
Same rule applies for all components in this inspector.